packages/dx-react-chart/src/plugins/zoom-pan.tsx

```diff
@@ -18,9 +18,55 @@ import {
   ScalesCache,
 } from '@devexpress/dx-chart-core';
 import {
-  ZoomAndPanProps, ZoomAndPanState, NumberArray,
+  ZoomAndPanProps, ZoomAndPanState, NumberArray, ZoomPanProviderProps,
 } from '../types';
 
+const events = {
+  wheel: 'onWheel',
+  mousedown: 'onDown',
+  touchstart: 'onDown',
+  touchmove: 'onTouchMove',
+  touchend: 'onTouchEnd',
+};
+
+class ZoomPanProvider extends React.PureComponent<ZoomPanProviderProps> {
+  ref: Element | undefined = undefined;
+  componentDidMount() {
+    this.ref = this.props.rootRef.current!;
+    if (!this.ref) return;
+    this.detachDocumentEvents();
+    this.attachDocumentEvents();
+  }
+
+  attachDocumentEvents() {
+    Object.keys(events).forEach((el) => {
+      this.ref!.addEventListener(el, this.handler(el), { passive: false });
+    });
+  }
+
+  detachDocumentEvents() {
+    Object.keys(events).forEach((el) => {
+      this.ref!.removeEventListener(el, this.handler(el));
```

Comment:
Created handlers should be persisted in the instance - now `handler` creates a new instance on every call.